### ISSUE TITLE
immediate error with any URL

### ISSUE THREAD

**Issue by coding-horror** [ludios/grab-site#1]
I just installed this on Ubuntu 14.04 x64 LTS per the install instructions, and it seemed to install with no errors.

But when I attempt to `./grab-site URL` I get this exception

```
 File "/root/.local/bin/wpull3", line 3, in <module>
    wpull.__main__.main()
  File "/root/.local/lib/python3.4/site-packages/wpull/__main__.py", line 29, in main
    builder.build()
  File "/root/.local/lib/python3.4/site-packages/wpull/builder.py", line 204, in build
    url_table = self._build_url_table()
  File "/root/.local/lib/python3.4/site-packages/wpull/builder.py", line 658, in _build_url_table
    'URLTableImplementation', path=self._args.database)
  File "/root/.local/lib/python3.4/site-packages/wpull/factory.py", line 60, in new
    instance = self._class_map[name](*args, **kwargs)
  File "/root/.local/lib/python3.4/site-packages/wpull/database/sqltable.py", line 240, in __init__
    DBBase.metadata.create_all(self._engine)
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/sql/schema.py", line 3404, in create_all
    tables=tables)
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1615, in _run_visitor
    with self._optional_conn_ctx_manager(connection) as conn:
  File "/usr/lib/python3.4/contextlib.py", line 59, in __enter__
    return next(self.gen)
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1608, in _optional_conn_ctx_manager
    with self.contextual_connect() as conn:
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/engine/base.py", line 1798, in contextual_connect
    self.pool.connect(),
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/pool.py", line 348, in connect
    return _ConnectionFairy._checkout(self, self._threadconns)
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/pool.py", line 644, in _checkout
    fairy = _ConnectionRecord.checkout(pool)
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/pool.py", line 440, in checkout
    rec = pool._do_get()
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/pool.py", line 873, in _do_get
    c = self._create_connection()
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/pool.py", line 285, in _create_connection
    return _ConnectionRecord(self)
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/pool.py", line 411, in __init__
    self.connection = self.__connect()
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/pool.py", line 538, in __connect
    connection = self.__pool._creator()
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/engine/strategies.py", line 90, in connect
    return dialect.connect(*cargs, **cparams)
  File "/root/.local/lib/python3.4/site-packages/sqlalchemy/engine/default.py", line 377, in connect
    return self.dbapi.connect(*cargs, **cparams)
TypeError: 't' is an invalid keyword argument for this function
```

Am I missing a dependency? 

It does create a subfolder for the URL with dupes_db, ignores, ignore_sets, and wpull.log (empty).

**Comment by techAPJ** [comment]
:+1: I can repro this.

If the querystring is like `?t=12345`, it exits with error `TypeError: 't' is an invalid keyword argument for this function`.

**Comment by ivan** [comment]
Thanks for the report, this is now fixed in master.

I've used grab-site to crawl a lot of websites, but never with a `?` in the starting URL, because `grab-site` is designed only for recursive crawls, and the starting URL rarely needs to be anything but the root of the site, or some `/subdirectory/` on the site.

Note that if you give grab-site a URL like `http://domain/forum?t=10`, it will walk up to the nearest `/` and, in this case, use `http://domain/` as the base of the recursive crawl.   Thus the entire domain will be crawled.
